bin/check-load.rb

```diff
@@ -8,7 +8,7 @@
 #   plain text
 #
 # PLATFORMS:
-#   Linux
+#   Linux, Windows, BSD, Solaris, etc
```

Comment:
I just copied that from https://github.com/sensu-plugins/sensu-plugins-load-checks/blob/master/bin/metrics-load.rb#L24, I'll take out Windows though.